src/test/java/bio/terra/common/CloudPlatformWrapperTest.java

```diff
@@ -0,0 +1,29 @@
+package bio.terra.common;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import bio.terra.common.category.Unit;
+import bio.terra.model.CloudPlatform;
+import java.util.Map;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+@Tag(Unit.TAG)
+class CloudPlatformWrapperTest {
+
+  @Test
+  void choose_Gcp() {
```

Comment:
Another way to write this is to have a single test that's parameterized on cloud platform and the test result. That would work because the `Map` is the same in both cases.